test/documentation/bio_doxygen_cfg.in

```diff
@@ -0,0 +1,101 @@
+PROJECT_NAME           = "b.i.o"
+PROJECT_BRIEF          = "The biological C++ library for Input and Output."
+PROJECT_LOGO           = ${BIO_DOXYGEN_SOURCE_DIR}/test/documentation/seqan_logo.png
+PROJECT_NUMBER         = ${BIO_VERSION}
+LAYOUT_FILE            = ${BIO_DOXYGEN_SOURCE_DIR}/test/documentation/DoxygenLayout.xml
+
+## PATHS
+OUTPUT_DIRECTORY       = ${BIO_DOXYGEN_OUTPUT_DIR}
+INCLUDE_PATH           = ${BIO_DOXYGEN_SOURCE_DIR}/include
+INPUT                  = ${BIO_DOXYGEN_SOURCE_DIR}/include               \
+                         ${BIO_DOXYGEN_SOURCE_DIR}/doc                   \
+                         ${BIO_DOXYGEN_SOURCE_DIR}/LICENSE
+##                       ${BIO_DOXYGEN_SOURCE_DIR}/CHANGELOG.md          \
+##                       ${BIO_DOXYGEN_SOURCE_DIR}/CODE_OF_CONDUCT.md    \
+##                       ${BIO_DOXYGEN_SOURCE_DIR}/CONTRIBUTING.md
+STRIP_FROM_PATH        = ${BIO_DOXYGEN_SOURCE_DIR}/include
+STRIP_FROM_INC_PATH    = ${BIO_DOXYGEN_SOURCE_DIR}/include
+EXAMPLE_PATH           = ${BIO_DOXYGEN_SOURCE_DIR}
+IMAGE_PATH             = ${BIO_DOXYGEN_SOURCE_DIR}/doc \
+                         ${BIO_DOXYGEN_SOURCE_DIR}/test/documentation
+
+## DOT SUPPORT
+HAVE_DOT               = ${BIO_DOXYGEN_HAVE_DOT}
+DOT_NUM_THREADS        = ${BIO_DOXYGEN_DOT_NUM_THREADS}
+DOT_IMAGE_FORMAT       = svg
+DOT_GRAPH_MAX_NODES    = 500
+INTERACTIVE_SVG        = ${BIO_DOXYGEN_HAVE_DOT}
+
+## MISC OPTIONS
+GENERATE_LATEX         = NO
+USE_MATHJAX            = ${BIO_DOXYGEN_USE_MATHJAX}
+HTML_TIMESTAMP         = YES
+EXT_LINKS_IN_WINDOW    = YES
+RECURSIVE              = YES
+DISABLE_INDEX          = YES
+GENERATE_TREEVIEW      = YES
+INLINE_INHERITED_MEMB  = NO
+COLLABORATION_GRAPH    = NO
+TEMPLATE_RELATIONS     = YES
+ENABLE_PREPROCESSING   = YES
+MACRO_EXPANSION        = YES
+EXPAND_ONLY_PREDEF     = YES
+SKIP_FUNCTION_MACROS   = NO
+SORT_BRIEF_DOCS        = YES
+TOC_INCLUDE_HEADINGS   = 2
+HTML_EXTRA_STYLESHEET  = ${BIO_DOXYGEN_SOURCE_DIR}/test/documentation/bio.css
+HTML_FOOTER            = ${BIO_DOXYGEN_OUTPUT_DIR}/bio_footer.html
```

Comment:
and this one, too

Reply:
this can stay the same even if we import the footer. cmake converts the `.in` file to `${PROJECT}_footer.html`